packages/extension-ui/src/Popup/Export.tsx

```diff
@@ -58,10 +62,15 @@ function Export ({ match: { params: { address } } }: Props): React.ReactElement<
         showBackArrow
         text={t<string>('Export account')}
       />
-      <div>
+      <div className={className}>
         <Address address={address}>
-          <MovedWarning danger>{t<string>("You are exporting your account. Keep it safe and don't share it with anyone.")}</MovedWarning>
-          <ActionArea>
+          <Warning
+            className='movedWarning'
+            isDanger
```

Comment:
That was a bug `danger` -> `isDanger`

Reply:
Cool! Love the drive-by bugs being fixed.